directives/questionnaire.py

```diff
@@ -47,6 +47,7 @@ class Questionnaire(ConfigurableExercise):
         'reveal-model-solutions': directives.unchanged,
         'grading-mode': directives.unchanged,
         'autosave': directives.flag,
+        'disable-duplicate-check': directives.flag,
```

Comment:
If this option is identical for the questionnaire and submit directives, then could the option be defined in the super class `ConfigurableExercise`? Well, it doesn't fit under the name and purpose of `ConfigurableExercise` and there are other old options that have been duplicated for questionnaires and submit exercises. Let's keep it like this then.